### PR TITLE
08-Calling Your Api Tutorial

### PR DESCRIPTION
**Pull request by tenmaster**
Added Calling your Api Tutorial
Update index.iml

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/android/08-calling-apis.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
+
+### Before Starting
+
+You should know already how to manage the `Credentials` object, as explained in [Session Management](03-session-handling.md) tutorial.
```
Comment (volbap) [inline articles/native-platforms/android/08-calling-apis.md]: You should already know

File: articles/native-platforms/android/08-calling-apis.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                }, new Response.ErrorListener() {
+
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+
+                    }
+                });
+```
+
+The customized AuthorizationRequestObject looks like:
```
Comment (volbap) [inline articles/native-platforms/android/08-calling-apis.md]: backtick the `AuthorizationRequestObject`

File: articles/native-platforms/android/08-calling-apis.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        public Map getHeaders() throws AuthFailureError {
+            Map headers = new HashMap();
+            headers.put("Bearer \\"+headerTokenID, "Authorization");
+            return headers;
+        }
+
+    }
+```
+	
+> This customized request is meant to manipulate the header of the JsonObjectRequest.
```
Comment (volbap) [inline articles/native-platforms/android/08-calling-apis.md]: backtick the `JsonObjectRequest`

File: articles/native-platforms/android/08-calling-apis.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+---
+title: Call you API
```
Comment (volbap) [inline articles/native-platforms/android/08-calling-apis.md]: Title should be `Calling APIs`

File: articles/native-platforms/android/08-calling-apis.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+### 2. Attach the token
+
+First you need to prepare the request:
+
+```java
+RequestQueue queue = Volley.newRequestQueue(this);
+String url = "YOUR API URL";
+```
+
+Then need to add the token as a mean of authentication to the request header. In this example we use Android's `Volley` and a custom `JsonObjectRequest`.
```
Comment (volbap) [inline articles/native-platforms/android/08-calling-apis.md]: Then, you need to add...

**Comment by volbap** [comment]
+1 @mpaktiti

File: articles/native-platforms/android/08-calling-apis.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
+---
+title: Calling APIs
+description: This tutorial will show you how to use the Auth0 tokens to make authenticated API calls.
+seo_alias: android
+---
+
+<%= include('../../_includes/_github', {
```
Comment (mpaktiti) [inline articles/native-platforms/android/08-calling-apis.md]: @tenmaster Please move this underneath System Requirements.

File: articles/native-platforms/android/08-calling-apis.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+	public void onErrorResponse(VolleyError error) {
+
+	}
+});
+```
+
+The customized `AuthorizationRequestObject` looks like:
+
+```java
+	public class AuthorizationRequestObject extends JsonObjectRequest
```
Comment (mpaktiti) [inline articles/native-platforms/android/08-calling-apis.md]: @tenmaster Please remove excessive tabs at the beginning of lines

File: articles/native-platforms/android/08-calling-apis.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+	
+Notice that how you configure your authorization header should match the standards that you're using in your API, this is just an example of what it could look like.
+	
+       
+### 3. Send the request
+
+At this point, you only need to schedule the request.
+
+```java
+		// Add the request to the RequestQueue.
```
Comment (mpaktiti) [inline articles/native-platforms/android/08-calling-apis.md]: @tenmaster Please remove excessive tabs at the beginning of lines

**Pull request closed without merging**